Makefile

```diff
@@ -14,3 +14,28 @@ $(MAKEFILE):
 	git clone --quiet --branch $(CI_VERSION) --depth 1 $(CI_REPOSITORY) $(CI_PATH);
 
 -include $(MAKEFILE)
+
+LD_FLAGS_INTEGRATION = -X main.version="integration-testing" -X main.build=$(BUILD) -X main.commit=$(COMMIT)
+
+GOTEST_INTEGRATION_TAGS_LIST = integration
+GOTEST_INTEGRATION_TAGS = $(GOTEST_INTEGRATION_TAGS_LIST)
+GOTEST_INTEGRATION = $(GOTEST) -parallel 1 -count 1 -tags="$(GOTEST_INTEGRATION_TAGS)"
+
+INTEGRATION_TEST_BUILD_PATH = "build-integration"
+INTEGRATION_TEST_BIN_PATH = $(INTEGRATION_TEST_BUILD_PATH)/bin
+
+GOBUILD_INTEGRATION = $(GOCMD) build -ldflags "$(LD_FLAGS_INTEGRATION)" -tags "$(GOTEST_INTEGRATION_TAGS)"
+
+clean-integration:
+	rm -rf $(INTEGRATION_TEST_BUILD_PATH)
+build-integration: BUILD_PATH=$(INTEGRATION_TEST_BUILD_PATH)
+build-integration: BIN_PATH=$(INTEGRATION_TEST_BIN_PATH)
+build-integration: GOBUILD=$(GOBUILD_INTEGRATION)
+build-integration: build
```

Comment:
why do you need repeated target here?

Reply:
Because I want to set some env vars before running `build`. This is the only way I found, maybe there's another way. I found it [here](https://stackoverflow.com/questions/15229833/set-environment-variable-inside-target).